tests/foreman/cli/test_repository.py

```diff
@@ -2105,7 +2108,27 @@ def test_negative_update_red_hat_repo():
 
         :expectedresults: hammer returns error code. The repository is not updated.
         """
-        pass
+        repo_set_list = RepositorySet.list({'organization-id': module_manifest_org.id})
+
+        RepositorySet.enable(
+            {
+                'organization-id': module_manifest_org.id,
+                'basearch': "x86_64",
+                'id': repo_set_list[0]['id'],
```

Comment:
This `id` is referenced multiple times via this index+key, consider assigning it directly, since nothing else is read from `repo_set_list`.

```
rh_repo_id = RepositorySet.list({'organization-id': module_manifest_org.id})[0]['id']
```

Reply:
Yes, got you point.

Reply:
In case of any confusion - suggested changes like this (`consider _______`) aren't required, especially when they come with a approved review. Thanks for making the update!